restler/utils/logger.py

```diff
@@ -671,9 +773,18 @@ def add_hash(replay_filename):
                 bug_bucket = bug_buckets[bucket_class][seq_hash]
                 bucket_hash = f"{seq_hash}_{bucket_class}"
                 name_header = bucket_class
+                checkerName = bucket_class[0:0+ bucket_class.find("_"+bucket_class.split("_")[len(bucket_class.split("_"))-1])]
```

Comment:
>bucket_class[0:0+ bucket_class.find("_"+bucket_class.split("_")[len(bucket_class.split("_"))-1])] [](http://example.com/codeflow?start=30&length=97)

This logic may not work in all cases (see `_get_bucket_origin` in bug_bucketing.py).  It might be simplest to add 'origin' directly to the BugBucket class and save it in the `update_bug_buckets` function when creating the bug bucket, then retrieving it here (`checker_name =  bug_bucket.origin`)

Reply:
updated as requested.